copilot/core/vectordb_utils.py

```diff
@@ -111,7 +113,7 @@ def get_text_splitter(ext):
     elif ext in ["json"]:
         return CopilotRecursiveJsonSplitter(max_chunk_size=300)
     elif ext in ["java"]:
-        return RecursiveCharacterTextSplitter.from_language(language=Language.JAVA)
+        return None
```

Comment:
Is this change correct?

Reply:
In the demo meeting we had defined some parameters for the max chunk size, I have added them.